jdi-light-angular-tests/pom.xml

```diff
@@ -2,49 +2,43 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <artifactId>jdi-2</artifactId>
-        <groupId>com.epam.jdi</groupId>
-        <version>1.1.16</version>
-    </parent>
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>jdi-light-angular-tests</artifactId>
+    <name>JDI Angular Tests</name>
+    <version>1.1.16</version>
```

Comment:
bump version to be equal jdi light version in master

Reply:
Done